hammerspoon/microphone.lua

```diff
@@ -0,0 +1,60 @@
+local message = require('status-message')
+
+local messageMuting = message.new('muted 🎤')
+local messageHot = message.new('hot 🎤')
+local last_mods = {}
+local recently_clicked = false
+local second_click = false
+
+display_status = function ()
+  -- Check if the active mic is muted
+  if hs.audiodevice.current(true).device:muted() then
+    messageMuting:notify()
+  else
+    messageHot:notify()
+  end
+end
+display_status()
+
+toggle = function (device)
```

Comment:
Tiny style-related request: Remove the whitespace between `function` and `(` for consistency with the existing code.